canonical-web-and-design/ubuntu.com
# GCP pages

## Done

GCP pages:
[ubuntu.com/gcp](https://docs.google.com/document/d/1Gvt9tNFDZkqkBcP0k79C6ueTJCmNZvwoXQoU19WEa8E/edit#heading=h.c6wjbvvp8mcm)
[ubuntu.com/gcp/pro](https://docs.google.com/document/d/1zR1m6CH1POU5tnzNzgNYanG2_EGQtKVNR3_dPew1pf4/edit#heading=h.n1426k6lmjxx)

*Note*: [/16-04/gcp](https://docs.google.com/document/d/14MeGfBfAiNxnKgJpTXhTEQpKMDAwoNp7Dqh0uqI7Dyc/edit) not included, as I believe this is more relevant to include it in a /16-04 PR

## QA

Check these pages
- [ ] https://ubuntu-com-9398.demos.haus/gcp
- [ ] https://ubuntu-com-9398.demos.haus/gcp/pro

Against copy docs:
- [ubuntu.com/gcp](https://docs.google.com/document/d/1Gvt9tNFDZkqkBcP0k79C6ueTJCmNZvwoXQoU19WEa8E/edit#heading=h.c6wjbvvp8mcm)
- [ubuntu.com/gcp/pro](https://docs.google.com/document/d/1zR1m6CH1POU5tnzNzgNYanG2_EGQtKVNR3_dPew1pf4/edit#heading=h.n1426k6lmjxx)
Check links are correct
Check meta descriptions
Check copy doc links
Check no-index is present
Check form id = 3860

## Issue / Card

Fixes #9397
